docs/releasehistory.rst

```diff
@@ -330,9 +353,9 @@ Tests added
   :py:meth:`Forcefield.create_openmm_system <openforcefield.typing.engines.smirnoff.forcefield.ForceField.create_openmm_system>`
```

Comment:
This is mangled (missing extra ` I think?)

![image](https://user-images.githubusercontent.com/7935382/85232877-4626cb00-b3c8-11ea-895c-37bf0888496a.png)